src/ReviewService.Abstractions/ReviewSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// The review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettings
+{
+	public int PrimaryActionCompletedCount { get; init; }
+
+	public int SecondaryActionCompletedCount { get; init; }
+
+	public int ApplicationLaunchCount { get; init; }
+
+	public DateTimeOffset? ApplicationFirstLaunched { get; init; }
```

Comment:
Maybe I should've spot it earlier, but shouldn't we have xml doc here

Reply:
I think the name is clear enough.